packages/bpk-component-image/src/BpkImageCornerStyles.js

```diff
@@ -0,0 +1,23 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016-2019 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+
+const BORDER_RADIUS_STYLES = { none: null, sm: 'sm' };
```

Comment:
Sorry I just noticed something else. This shouldn't evaluate to `null`:

```suggestion
const BORDER_RADIUS_STYLES = { none: 'none', sm: 'sm' };
```